droidlet/perception/semantic_parsing/tests/test_y_print_parsing_report.py

```diff
@@ -1185,14 +1186,23 @@ def compare_full_dictionaries(d1, d2):
         return compare_dicts(d1, d2)
 
 
+@pytest.mark.usefixtures("flag_m")
 class TestDialogueManager(unittest.TestCase):
     def setUp(self):
         opts = MockOpt()
         opts.nsp_data_dir = TTAD_BERT_DATA_DIR
         opts.ground_truth_data_dir = GROUND_TRUTH_DATA_DIR
         opts.nsp_models_dir = TTAD_MODEL_DIR
         opts.no_ground_truth = False
-        self.chat_parser = NSPQuerier(opts=opts)
+        if self.flag_m == "True":
```

Comment:
sorry I don't know the `pytest.mark.usefixtures` very well, can we use the following here:
`opts.flat_m = self.flag_m`?

Reply:
Thanks @snyxan , I agree with this comment as well

Reply:
I update the type of flag to store_true. Thanks for comment.